### ISSUE TITLE
Improve accesibility

### ISSUE THREAD

**Issue by bogas04** [KhalisFoundation/sttm-web#336]
We should make STTM defacto place for differently abled Sikhs to access Baani. 

* [ ] Cleaner & semantic markup.
* [ ] [alt] for `<img />`
* [ ] Non `<button />` buttons should have `[role="button"][aria-label="open/hide/show/close"]`
* [ ] ...

**Comment by gsin11** [comment]
Basic handling of accessibility is in place now by adding the following stuff:
- Added alt attributes to images.
- Added aria-labels to empty buttons.

Ticket ID: https://github.com/KhalisFoundation/sttm-web/issues/1400

What is pending?
We can create another ticket to fix component-specific accessibility (example: right sidebar).
